### DISCUSSION TITLE
Qiitaの説明には、ユーザーに寄り添った親切なあと一言が足りないと思います

### DISCUSSION THREAD

**Discussion by tsubolabo** [increments/qiita-discussions#459] [ヘルプ]

**Comment by tanaka-asami415** [comment]
@tsubolabo 
ありがとうございます🙏

> 一方、「ベータ版切り替え」のページには、「ベータ版を有効化するとヘッダーのデザインが変わります。」とあります。真黒な帯はベータ版だけに表示され、正式バージョンでは緑に戻るのでしょうか？

はい。ベータ版の時はベータ版を利用されていることが分かりやすいよう、ヘッダーを黒色にしております。
ベータ―版として公開されたデザイン画像なため、ヘッダーが黒色になっているのですが
確かに初めての方には紛らわしいかもしれませんね…

今後は紛らわしい際には、注釈として以下のような文章を入れるようにします！🙇
**※上記はベータ版の画像のため、ヘッダーが黒になっています。正式公開の際は通常色になります。**

取り急ぎ、以下のblog記事を修正いたしました！
https://blog.qiita.com/editor-design-update-beta/
